control-plane-operator/controllers/hostedcontrolplane/assets/cluster-bootstrap/cluster-csr-approver-clusterrole.yaml

```diff
@@ -0,0 +1,38 @@
+# https://github.com/openshift/cluster-kube-controller-manager-operator/blob/release-4.9/bindata/assets/kube-controller-manager/csr_approver_clusterrole.yaml
```

Comment:
this clusterrole is used for cluster-policy-controller to approve monitoring csr